Cargo.toml

```diff
@@ -29,19 +29,13 @@ links = ["egui-winit/links"]
 
 
 [dependencies]
-egui = { version = "0.26.2", default-features = false, features = [
-  "bytemuck",
-  "default_fonts"
```

Comment:
It seems like the `pure_glium` example is broken at least on Wayland. The fonts are missing. You need to enable the `default_fonts` feature of `egui` for the example to work correctly. Though preferably this feature would only be enabled by default in dev.